### PR TITLE
Unify log format on CC

### PR DESCRIPTION
**Pull request by arukuka**
# Summary

- Unify log format
- Load log level from environment variables

# Purpose

- Unify log format
- Find out location where logger is called
- Make log level variable

# Contents

- Change interface of QuickMPC Logger
- Add macro to find called location
- Use QuickMPC logger macro instead of calling `spdlog` directly
- Load log level: `LOG_LEVEL` from environment variables

# Testing Methods Performed

- Ran demo
- Ran CC small, medium test

### CODE REVIEW

File: src/ComputationContainer/LogHeader/BUILD [hunk 1]
```diff
@@ -7,5 +7,5 @@ cc_library(
     deps = [
         "@libspdlog//:spdlog"
     ],
-    visibility = ["//visibility:public"],
+    visibility = ["//Logging:__pkg__"],
```
Comment (arukuka) [inline src/ComputationContainer/LogHeader/BUILD:10]:
```suggestion
    visibility = ["//visibility:private"],
```

and move to Logging/ package

File: src/ComputationContainer/Logging/Logger.hpp [hunk 2]
```diff
@@ -1,92 +1,139 @@
...
-        {Log::LogLevel::Info, "INFO"},
-        {Log::LogLevel::Debug, "DEBUG"},
-        {Log::LogLevel::Error, "ERROR"}};
-
 private:
-    std::string getTime() const
+    static constexpr auto LOG_FORMAT =
+        std::string_view("%Y-%m-%d %T %z | %^%-5l%$ | %g:%!:%# - %v");
+    std::shared_ptr<spdlog::sinks::dist_sink_mt> sinks;
+    std::shared_ptr<spdlog::logger> logger;
```
Comment (arukuka) [inline src/ComputationContainer/Logging/Logger.hpp:34]:
```suggestion
    std::unique_ptr<spdlog::sinks::dist_sink_mt> sinks;
    std::unique_ptr<spdlog::logger> logger;
```

File: src/ComputationContainer/Logging/Logger.hpp [hunk 3]
```diff
@@ -1,92 +1,139 @@
...
+    Log()
+        : sinks(std::make_shared<spdlog::sinks::dist_sink_mt>())
+        , logger(std::make_shared<spdlog::logger>("QMPC Logger", sinks))
```
Comment (arukuka) [inline src/ComputationContainer/Logging/Logger.hpp:37]:
```suggestion
        : sinks(std::make_unique<spdlog::sinks::dist_sink_mt>())
        , logger(std::make_unique<spdlog::logger>("QMPC Logger", sinks))
```

File: src/ComputationContainer/Logging/Logger.hpp (hunk 2 above)
Comment (arukuka) [reply src/ComputationContainer/Logging/Logger.hpp]: Changed only `logger` to unique ptr.  
See 3263e64a2014d77c08d4adb75e372e435f6c8fd7 more details.

File: src/ComputationContainer/Logging/Logger.hpp (hunk 3 above)
Comment (arukuka) [reply src/ComputationContainer/Logging/Logger.hpp]: Changed only logger to unique ptr.
See https://github.com/acompany-develop/QuickMPC/commit/3263e64a2014d77c08d4adb75e372e435f6c8fd7 more details.

File: src/ComputationContainer/Logging/Logger.hpp (hunk 2 above)
Comment (mdonaka) [reply src/ComputationContainer/Logging/Logger.hpp]: I have confirmed it. Thank you!

**Review by mdonaka — APPROVED** [review]

**Pull request merged**